missing-values-imputer/wind-imputation/preprocess.py

```diff
@@ -0,0 +1,31 @@
+from ydata.dataset import Dataset
+
+def preprocess_data(df: Dataset):
```

Comment:
This definition also appears in the Preprocess notebook. Should we keep it in only one place and import it in other instances?

Reply:
You saw the other one, they're slightly different , I'm going to rename one of them